src/components/dashboard/navigation.js

```diff
@@ -0,0 +1,100 @@
+import React, { Component } from "react";
+import { NavLink } from "react-router-dom";
+import {
+  faSearch,
+  faUserCircle,
+  faAngleDown,
+  faBars
+} from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import "../dashboard/_Navigation.scss";
+import { faBell, faQuestionCircle } from "@fortawesome/free-regular-svg-icons";
+
+import { ifDev } from "../../utils/removeAttribute.js";
+import RegisterNavLinks from "../layout/nav-layouts/RegisterNavLinks";
+
+class Navigation extends Component {
+  state = {
+    search: "",
+    pathname: ""
```

Comment:
This is the state I am using to look at the window's url and find out which page the user is currently on and update the nav bar based on that.